integration-tests/deployment/keystone/state.go

```diff
@@ -0,0 +1,79 @@
+package keystone
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/capabilities_registry"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/forwarder"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/ocr3_capability"
+)
+
+type GetContractSetsRequest struct {
+	Chains      map[uint64]deployment.Chain
+	AddressBook deployment.AddressBook
+}
+
+type GetContractSetsResponse struct {
+	ContractSets map[uint64]ContractSet
+}
+
+type ContractSet struct {
```

Comment:
for a follow up, when you want to generate "RDD style" data you can follow the "view" package pattern we did here https://github.com/smartcontractkit/chainlink/tree/815b2c80ff6b0dff940497f7734aff3662e7b4f9/integration-tests/deployment/ccip/view  (versioned, json marshallable structs) generated from this ContractSet